### PR TITLE
feat: init portal app

### PR DESCRIPTION
**Pull request by matthieu-crouzet**
## Proposed change

Creation of the portal (no content in the app)

## Related issues

- :rocket: Feature

### CODE REVIEW

File: apps/portal/cms.json
Comment (mrednic-1A) [inline apps/portal/cms.json:1]: IMO you can remove this file, and any other files/features related to CMS

File: apps/portal/src/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'app',
+  templateUrl: './app.template.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
```
Comment (mrednic-1A) [inline apps/portal/src/app/app.component.ts:7]: _OPEN QUESTION_ Should we do this app with standlone components only ?

File: apps/portal/tsconfig.dev.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.build",
+"angularCompilerOptions": {
```
Comment (cpaulve-1A) [inline apps/portal/tsconfig.dev.json:3]: NB: incorrect indent

File: packages/@ama-sdk/generator-sdk/src/generators/shell/templates/base/package.json [hunk 3]
```diff
@@ -73,7 +73,7 @@
     "@angular-devkit/schematics-cli": "~15.1.3",
     "@angular-eslint/eslint-plugin": "~15.2.0",
     "@angular-eslint/eslint-plugin-template": "~15.2.0",
-    "@angular-eslint/utils": "~15.2.0",
+    "@angular-eslint/utils": "^15.2.0",
```
Comment (cpaulve-1A) [inline packages/@ama-sdk/generator-sdk/src/generators/shell/templates/base/package.json:76]: why do you change this dependency?

**Pull request closed without merging**